OSPatching/README.md

```diff
@@ -1,27 +1,25 @@
 # OSPatching Extension
-Allow the owner of the Azure VM to configure the Linux VM patching schedule
-cycle. And the actual patching operation is automated based on the
-pre-configured schedule.
+Allows the owner of the Azure VM to configure a Linux VM patching schedule cycle
+or perform OS patching on-demand as a one-time task. The actual patching operation
+is scheduled as a cron job.
 
-Lastest version is 2.0.
+Lastest version is 2.3.
 
-You can read the User Guide below.
-* [Automate Linux VM OS Updates Using OSPatching Extension (outdated, needs to update)](http://azure.microsoft.com/blog/2014/10/23/automate-linux-vm-os-updates-using-ospatching-extension/)
+You can read the User Guide, [Automate Linux VM OS Updates Using OSPatching Extension (outdated, needs to update)](http://azure.microsoft.com/blog/2014/10/23/automate-linux-vm-os-updates-using-ospatching-extension/).
 
 OSPatching Extension can:
 * Patch the OS automatically as a scheduled task
-* Patch the OS automatically as a one-off
-* it can be stopped before the actual patching operation
-* the status of VM can be checked by user-defined scripts,
-which can be stored locally, in github or Azure Storage
+* Patch the OS as a one-time task
+* The patching can be stopped before the actual patching operation begins
+* The status of VM can be checked by user-defined scripts stored locally, in GitHub, or in Azure Storage
 
 # User Guide
 
 ## 1. Configuration schema
+All settings are set in the protected configuration. No settings are available in the public configuration and it can be omitted.
```

Comment:
This is a minor change (2.0 to 2.3), but this is a breaking change.  I would not do this.  You should support both modes unless there's a need to break customers.